### PR TITLE
Add nervos network

### PR DESCRIPTION
**Pull request by antiyro**
Added Nervos network testnet/mainnet to sdk chain.ts

### CODE REVIEW

File: src/chains.ts [hunk 1]
```diff
@@ -1195,6 +1195,44 @@ export const OASIS_TESTNET: Chain = {
...
+  id: 'godwoken_mainnet',
+  chain_id: 71394,
+  name: 'Godwoken Testnet',
+  symbol: 'CKB',
+  mainnet: true,
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/godwoken.png',
+  pangolin_is_live: false,
+  tracked_by_debank: false,
+  supported_by_gelato: false,
+  rpc_uri: 'https://mainnet.godwoken.io/rpc',
```
Comment (HariSeldon23) [inline src/chains.ts:1208]: This should be https://mainnet.godwoken.io/rpc/eth-wallet

File: src/chains.ts [hunk 2]
```diff
@@ -1195,6 +1195,44 @@ export const OASIS_TESTNET: Chain = {
   blockExplorerUrls: ['https://testnet.explorer.emerald.oasis.dev'],
 }
 
+export const GODWOKEN_MAINNET: Chain = {
+  id: 'godwoken_mainnet',
+  chain_id: 71394,
+  name: 'Godwoken Testnet',
+  symbol: 'CKB',
+  mainnet: true,
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/godwoken.png',
```
Comment (HariSeldon23) [inline src/chains.ts:1204]: This URL isn't found

File: src/chains.ts [hunk 3]
```diff
@@ -1195,6 +1195,44 @@ export const OASIS_TESTNET: Chain = {
...
+  pangolin_is_live: false,
+  tracked_by_debank: false,
+  supported_by_gelato: false,
+  rpc_uri: 'https://mainnet.godwoken.io/rpc',
+  nativeCurrency:{
+    name: 'Nervos Network',
+    symbol: 'CKB',
+    decimals: 18,
+  },
+  blockExplorerUrls: ['https://layerview.io'],
```
Comment (HariSeldon23) [inline src/chains.ts:1214]: This should be 
https://gwscan.com

File: src/chains.ts [hunk 4]
```diff
@@ -1195,6 +1195,44 @@ export const OASIS_TESTNET: Chain = {
...
+  id: 'godwoken_mainnet',
+  chain_id: 71393,
+  name: 'Godwoken Testnet',
+  symbol: 'CKB',
+  mainnet: false,
+  logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/godwoken.png',
+  pangolin_is_live: false,
+  tracked_by_debank: false,
+  supported_by_gelato: false,
+  rpc_uri: 'https://godwoken-testnet-web3-rpc.ckbapp.dev',
```
Comment (HariSeldon23) [inline src/chains.ts:1227]: Not sure this is correct. Need to confirm it's not x/rpc/eth-wallet like Godwoken is

File: src/chains.ts [hunk 5]
```diff
@@ -1195,6 +1195,44 @@ export const OASIS_TESTNET: Chain = {
...
+  nativeCurrency:{
+    name: 'Nervos Network',
+    symbol: 'CKB',
+    decimals: 18,
+  },
+  blockExplorerUrls: ['https://aggron.layerview.io'],
```
Comment (HariSeldon23) [inline src/chains.ts:1233]: Should be https://aggron.gwscan.com/en-US

**Review by HariSeldon23 — CHANGES_REQUESTED** [review]

File: src/chains.ts (hunk 4 above)
Comment (antiyro) [reply src/chains.ts:1227]: _L2 Godwoken Testnet explorer: https://v1.aggron.gwscan.com/_ seems to be this link.

Comment (antiyro) [reply src/chains.ts:1227]: https://startwithnervos.com/godwoken

File: src/chains.ts (hunk 5 above)
Comment (antiyro) [reply src/chains.ts:1233]: https://v1.aggron.gwscan.com/ it seems to be this link (https://startwithnervos.com/godwoken)

File: src/chains.ts (hunk 4 above)
Comment (antiyro) [reply src/chains.ts:1227]: https://godwoken-testnet-web3-v1-rpc.ckbapp.dev/ it seems to be this link from what I see here : https://startwithnervos.com/godwoken

**Review by HariSeldon23 — APPROVED** [review]

**Pull request merged**